google/cloud/bigquery/table.py

```diff
@@ -452,17 +453,20 @@ def schema(self):
                 instance or a compatible mapping representation of the field.
         """
         prop = self._properties.get(self._PROPERTY_TO_API_FIELD["schema"])
-        if not prop:
+        if not prop:  # if empty Schema, empty list, None
             return []
```

Comment:
Could we return an "empty" Schema here for consistency?

```suggestion
            return Schema()
```

Reply:
Along the lines of least surprise... if the user passes in an empty `list` or `None`, they will get back an empty `list`. If they pass in an empty `Schema()` they will get back an empty `Schema()`